tools/lsp/preview.rs

```diff
@@ -1012,8 +1012,10 @@ fn get_path_from_cache(path: &Path) -> Option<(SourceFileVersion, String)> {
 pub enum LoadBehavior {
     /// We reload the preview, most likely because a file has changed
     Reload,
-    /// We load the preview because the user asked for it. The UI should become visible if it wasn't already
+    /// Load the preview, but don't require focus.
```

Comment:
This does show the window though.
I guess the comment should be 
```suggestion
    /// Load the preview and make the window visible if it wasn't already.
```

Reply:
👍

Reply:
Ahh, fair point. I can rename it and remove the Load :)

Reply:
Ah, looking at the diff i see now that there is a difference between load and reload that load also resets the component (eg when clicked on a component from the library, which is behavior is not available anymore, but  could be brought back) so there is a use case for that option.

Reply:
I don't feel to strongly about it. Would you like me to remove the last commit again?